tests/test_utils/test_schemaview.py

```diff
@@ -503,6 +523,19 @@ def test_imports_from_schemaview(schema_view_with_imports: SchemaView) -> None:
     assert len(view.all_classes(imports=False)) == len(view2.all_classes(imports=False))
 
 
+@pytest.mark.parametrize(
+    ("schema", "imports_closure"),
+    [
+        ("schema_view_no_imports", {"kitchen_sink"}),
+        ("schema_view_with_imports", {"kitchen_sink", "core", "linkml:types"}),
+    ],
+)
+def test_imports_closure(schema: str, imports_closure: set[str], request: pytest.FixtureRequest) -> None:
```

Comment:
This test was hidden further down in the file and not as well articulated. I wanted to ensure my understanding of the function so improved the test.